checkov/sca_package_2/output.py

```diff
@@ -0,0 +1,364 @@
+from __future__ import annotations
```

Comment:
this is a copy of the existing one and then the other one will be deleted after the migration is done, right?

Reply:
yes